tf/tfprocess.py

```diff
@@ -487,14 +487,13 @@ def process_v2(self, batch_size, test_batches, batch_splits=1):
             self.manager.save()
             print("Model saved in file: {}".format(self.manager.latest_checkpoint))
             evaled_steps = steps.numpy()
-            leela_path = self.manager.latest_checkpoint + "-" + str(evaled_steps)
-            swa_path = self.manager.latest_checkpoint + "-swa-" + str(evaled_steps)
+            path = os.path.join(self.root_dir, self.cfg['name'])
```

Comment:
I don't see how this does what is intended. It would appear to me that while the run name is in the filename, its no longer in the path as well, so these weights files will be saved up one directory from where they used to be.

Reply:
Nevermind, I was too sleepy to recall that root_dir already contains the name.

However since checkpoints don't contain step count in their filename anymore - I think this is still a bit awkward to map checkpoint to file.  I think I would rather perform some other mutation of latest_checkpoint to add the run as part of the name.

Reply:
Changing the checkpoint filename seems a reasonable alternative - so resolved!